File: DataStructures/LinkedList/LinkedList.cs
class LinkedList {
    // The first node(head) of the linked list 
    // will be ab object of type Node (null by default)

    Node head;

    class Node {
        int data;
        Node next;

        //Constructor to create a new node
        Node(int d) {data = d;}
    }
}

File: DataStructures/LinkedList/LinkedList.py
# Node class
class Node:
    #Function to initialise the node object
    def __init__(self, data):
        self.data = data #assign data
        self.next = None # initialise as null

class LinkedList: 

    #Function to initialise the linked list object
    def __init__(self):
        self.head = None
